### ISSUE TITLE
Term colors does not work properly on windows

### ISSUE THREAD

**Issue by pravic** [regexident/cargo-modules#1]
Here is output:
```
←[1;32mPublic modules←[0m
←[1;33mPrivate modules←[0m
←[1;32msciter←[0m
←[1;34m ├── ←[0m←[1;33mcapi←[0m
←[1;34m │   ├── ←[0m←[1;33mscapi←[0m
←[1;34m │   ├── ←[0m←[1;33mscbehavior←[0m
←[1;34m │   ├── ←[0m←[1;33mscdef←[0m
←[1;34m │   ├── ←[0m←[1;33mscdom←[0m
←[1;34m │   ├── ←[0m←[1;33mscgraphics←[0m
←[1;34m │   ├── ←[0m←[1;33mschandler←[0m
←[1;34m │   ├── ←[0m←[1;33mscrequest←[0m
←[1;34m │   ├── ←[0m←[1;33msctiscript←[0m
←[1;34m │   ├── ←[0m←[1;33msctypes←[0m
←[1;34m │   └── ←[0m←[1;33mscvalue←[0m
←[1;34m ├── ←[0m←[1;32mdom←[0m
```

**Comment by regexident** [comment]
Hm, that's the color-escaped output.

I'm using the [colored](https://github.com/mackwic/colored) crate for coloring. Which [appears to lack Windows support](https://github.com/mackwic/colored#todo). :confused:

One (potential) way to get it working on Windows is to set the [`CLICOLOR`](http://bixense.com/clicolors/) environment variable to `0`:

    CLICOLOR=0 cargo modules

**Comment by pravic** [comment]
May be its better to disable color on windows by default?

**Comment by regexident** [comment]
Yes, I suppose. Will need some minor refactoring (to keep the code clean), but absolutely doable.

I'll probably make the colors optional and add a `--color` option to enable them.

I just opened a somewhat related issue you might want to take a look at:
https://github.com/regexident/cargo-modules/issues/2

**Comment by regexident** [comment]
Should be fixed with 0.2.1 now.
